microsoft-edge/extensions-chromium/getting-started/manifest-format.md

```diff
@@ -88,11 +91,15 @@ For reference information about each field, see [Manifest file format (V3)](http
     "page": "options.html"
   },
   "permissions": ["tabs"],
+  "permissions": ["sidePanel"],
   "platforms": ...,
   "replacement_web_app": ...,
   "requirements": {...},
   "sandbox": [...],
   "short_name": "Short Name",
+  "side_panel": {
```

Comment:
@Bhuvana-Priya - check the lines I added:
* dup "action"
* dup "permissions"
* new "side_panel"

what about the MV2 listing below -- make similar changes there?

Reply:
Outdated.  Newer [comment](https://github.com/MicrosoftDocs/edge-developer/pull/2648/files#r1239184115)